source/enterprise/tech-guide/installation-guide.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Shopware Enterprise Dashboard Installation Guide
+github_link: enterprise/tech-guide/installation-guide.md
+indexed: true
+---
+
+This guide will cover the Shopware Enterprise Dashboard installation process
+
+<div class="toc-list"></div>
+
+
+## Dependencies
+
+The following system libraries/application are required to install and run the Shopware Enterprise Dashboard from a release package:
+
+- Linux operating system
+- PHP 5.6 or later (PHP 7 recommended) including CLI support
+- Apache2 web server
+- MySQL
+- Ansible Version 1.9+
+- Beanstalk Version 1.9+
+- Supervisor 3.0+ 
+
+If you are setting up a development version of the EDB, you might need the following additional tools:
+- Ant
+- NodeJS + NPM
+  - Grunt
+  - Bower
+  - Jasmine
+- Vagrant + Virtualbox
+
+Similar architectures might also work, but are not officially supported.
+
+## Installing the Shopware Enterprise Dashboard
+
+### Using the installation package
+
+For evaluation, testing or production purposes, it's recommended using one of the available installation packages:
+- Download the installation package
+- Extract the archive to your web server's content folder
+- Enable web server access to _APPLICATION_/web
+- Execute `php setup.phar` through the command line
+- Follow the instructions
+- Copy _APPLICATION_/supervisord/edb.conf.dist to _APPLICATION_/supervisord/edb.conf and configure your `supervisor` to use it.
```

Comment:
link the *.dist file to the supervisor configuration directory - like sites-enabled vs sites-available